app/cdap/components/NamespaceAdmin/ServiceAccounts/EditConfirmDialog.tsx

```diff
@@ -88,6 +93,7 @@ export const EditConfirmDialog = ({
   const gcloudCommandParams = {
     identity: namespaceIdentity || undefined,
     gsaEmail: serviceAccountInputValue || undefined,
+    k8snamespace: namespacedCreationHookEnabled ? k8snamespace : 'default',
```

Comment:
There are two conditions when `namespaceCreationHook` is enabled and `k8snamespace` is present and not null or empty otherwise `default`

Reply:
yes, that is taken care. Verified with `k8snamespace` and works as expected, screen-shot added in the description is the same, PTAL.

Reply:
Empty value check was failing. Thanks for pointing out Ankit. Corrected it now .